alumni.ch.tudelft.nl/index.html@p=735_BBQ_2024.html

```diff
@@ -0,0 +1,519 @@
+<!DOCTYPE html>
+
+<!--[if IE 8]>
+<html id="ie8" lang="en-US">
+<![endif]-->
+<!--[if !(IE 8) ]><!-->
+<html lang="en-US">
+<!--<![endif]-->
+<head>
+<meta charset="UTF-8" />
+<meta name="viewport" content="width=device-width" />
+<link rel="profile" href="http://gmpg.org/xfn/11" />
+<link rel="pingback" href="https://alumni.ch.tudelft.nl/xmlrpc.php" />
+<title>Alumni BBQ - 31st of August 2024 &#8211; Constantijn Huygens</title>
+<link rel='dns-prefetch' href='https://fonts.googleapis.com/' />
+<link rel='dns-prefetch' href='https://s.w.org/' />
+<link rel="alternate" type="application/rss+xml" title="Constantijn Huygens &raquo; Feed" href="feed/index.html" />
+<link rel="alternate" type="application/rss+xml" title="Constantijn Huygens &raquo; Comments Feed" href="comments/feed/index.html" />
+<link rel="alternate" type="application/rss+xml" title="Constantijn Huygens &raquo; Alumni BBQ - 31th August Comments Feed" href="new-years-drink-14th-january/feed/index.html" />
+<!-- This site uses the Google Analytics by MonsterInsights plugin v7.13.3 - Using Analytics tracking - https://www.monsterinsights.com/ -->
```

Comment:
I believe Google Analytics is not allowed, since this conflicts with the privacy policy. Especially since we do not control this analytics account afaik. Please remove or get permission from the board to use this.

Reply:
Thanks Dany, I have removed all references to Google Analytics and Monsterinsights from the new HTML page. However since I simply copied an existing HTML page, I think the rest of the static site does still contain Google Analytics.
I will create a separate PR removing the other analytics code.